ddl/index.go

```diff
@@ -1081,7 +1081,11 @@ func onDropIndex(d *ddlCtx, t *meta.Meta, job *model.Job) (ver int64, _ error) {
 			// the partition ids were append by convertAddIdxJob2RollbackJob, it is weird, but for the compatibility,
 			// we should keep appending the partitions in the convertAddIdxJob2RollbackJob.
 			job.FinishTableJob(model.JobStateDone, model.StateNone, ver, tblInfo)
-			job.Args = append(job.Args, indexInfo.ID, getPartitionIDs(tblInfo))
+			if indexInfo.Global{
```

Comment:
Tips, if your editor/IDE does not format go code automatically, you can run `make fmt`
```suggestion
			if indexInfo.Global {
```